### DISCUSSION TITLE
Multiple target files from a single source

### DISCUSSION THREAD

**Discussion by glostis** [twpayne/chezmoi#2296] [Q&A] [answered]
Hi,

I would like for `chezmoi` to manage a single source file (let's call it `file.txt`) to multiple target files:
```
/home/user/path1/file.txt
/home/user/path2/file.txt
```

I guess there are several hacky ways to do this (for example, with a bash script that symlinks `/home/user/path1/file.txt` to `/home/user/path2/file.txt`, and have `chezmoi` manage only `/home/user/path1/file.txt`), but is there a "smarter" and "cleaner" way to do this?

Thanks!

**Comment by bradenhilton** [comment, marked as answer]
https://www.chezmoi.io/user-guide/manage-machine-to-machine-differences/#handle-different-file-locations-on-different-systems-with-the-same-contents

**Reply by glostis** [reply]
Thanks!
I had seen this documentation entry but had not realized that the solution to handle different file locations on different machines with the same contents can also handle different file locations on the same machine, which seems obvious in retrospect :smile:
